source/ios/AdaptiveCards/AdaptiveCards/AdaptiveCards/ACOHostConfig.mm

```diff
@@ -31,6 +31,17 @@ - (instancetype)initWithConfig:(std::shared_ptr<HostConfig> const &)config
     self = [super init];
     if(self && config){
         _config = config;
+        // check if requested font family name is supported by iOS, if so save it for future uses
+        NSString *requestedFontFamilyName = [NSString stringWithCString:_config->fontFamily.c_str() encoding:NSUTF8StringEncoding];
+        for(NSString* fontFamilyName in UIFont.familyNames){
+            if([fontFamilyName isEqualToString:requestedFontFamilyName]){
+                _fontFamilyName = fontFamilyName;
+            }
+        }
+        // if the requested font family name is not supported, use system font instead
+        if(!_fontFamilyName){
+            _fontFamilyName = @"-apple-system','HelveticaNeue";
```

Comment:
>-apple-system','HelveticaNeue [](start = 32, length = 29)

Don't put quotes in this string under the assumption that it will eventually be embedded into a larger quoted string. If you want to return multiple font families, it would be better to return an array of strings, then let the caller format them however it likes.

Reply:
got it, thanks

---
In reply to: [172292911](https://github.com/Microsoft/AdaptiveCards/pull/1207#discussion_r172292911) [](ancestors = 172292911)